src/backend/export/outputVendors/googleSheets/googleSheets.ts

```diff
@@ -44,6 +44,12 @@ const createTransactionsInGoogleSheets: ExportTransactionsFunction = async (
     transaction.category,
     transaction.accountNumber,
     transaction.hash,
+    moment(transaction.processedDate).format(GOOGLE_SHEETS_DATE_FORMAT),
+    transaction.identifier,
+    transaction.status,
+    transaction.type,
+    transaction.installments,
+    transaction.originalAmount,
     transaction.originalCurrency
```

Comment:
```suggestion
    transaction.hash,
    transaction.originalCurrency,
    moment(transaction.processedDate).format(GOOGLE_SHEETS_DATE_FORMAT),
    transaction.identifier,
    transaction.status,
    transaction.type,
    transaction.installments,
    transaction.originalAmount
```

I think you need to add new fields to the end (after `transaction.originalCurrency`).

Imagine an existing sheet, today the last column is `originalCurrency`. Now you are adding new data, that contains more fields, and the 8th column was `originalCurrency` and now becomes `processedDate`.

Reply:
no problem.
note that installments created an error. 
i believe it has something to do with formatting. for instance 3 installments out of 4 is 3/4 - there is a problem text vs number. 
anyway, meanwhile in my newer version i've removed the field installements.